app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i(create destroy)
+  before_action :correct_user, only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build micropost_params
+    if @micropost.save
+      flash[:success] = t ".success_create"
+      redirect_to root_path
+    else
+      @feed_items = []
+      render "static_pages/home"
```

Comment:
You have already the root page as "static_pages#home" in the route. So you can use root_path